dash/long_callback/managers/celery_manager.py

```diff
@@ -13,6 +13,8 @@
 
 
 class CeleryLongCallbackManager(BaseLongCallbackManager):
+    """Deprecated: use `import CeleryManager from dash` instead."""
```

Comment:
Can we give the primary class here the name we want (`CeleryManager`) and make `CeleryLongCallbackManager` into the subclass, defined right in this file? That would make it easier for people to read the code going forward, by removing a layer of indirection from the recommended usage.

Also the deprecation note should say `from dash import ...` not `import ... from dash` right?